### PR TITLE
F# yaml index page

### PR DESCRIPTION
**Pull request by cartermp**
Here's my first stab at it.

### CODE REVIEW

**Comment by BillWagner** [comment]
closing and reopening to get a fresh build.

File: docs/fsharp/index.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    linkLists:
+      - linkListType: get-started
+        links:
+          - text: "What is F#?"
+            url: what-is-fsharp.md
+          - text: "Install F#"
+            url: get-started/install-fsharp.md
+          - text: "Introductory tutorials"
+            url: tutorials/intro-to-csharp/index.md
+          - text: "Create C# apps with Visual Studio"
```
Comment (BillWagner) [inline docs/fsharp/index.yml]: This seems wrong. Is there an equivalent F# page?

**Review by BillWagner — COMMENTED** [review]
So far, just a first pass. I saw one issue now.

File: docs/fsharp/index.yml (hunk 1 above)
Comment (cartermp) [reply docs/fsharp/index.yml]: YOikes, will fix

**Comment by cartermp** [comment]
@BillWagner this is ready.

**Review by BillWagner — CHANGES_REQUESTED** [review]
I like the content @cartermp 

There are a number of build warnings because index.md => index.yml in the F# folder. Once those are fixed, this is ready to merge.

**Comment by cartermp** [comment]
@BillWagner done

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
This LGTM. You can :shipit: when ready

**Pull request merged**